### PR TITLE
Provide more details on how to upgrade a CAMO enabled cluster.

### PR DESCRIPTION
**Pull request by mwanner**
BDR-3002

## What Changed?

Provide more detailed instructions on how to upgrade a CAMO enabled cluster from BDR 3.7 or 4.x to BDR 5.0.

### CODE REVIEW

File: product_docs/docs/pgd/5/upgrades/index.mdx
Comment (djw-m) [inline product_docs/docs/pgd/5/upgrades/index.mdx:125]: Is the condition necessary here? Would simply "Create a dedicated node group" suffice.

(Looking to keep directions simple and as straight lined as possible)

Comment (mwanner) [reply product_docs/docs/pgd/5/upgrades/index.mdx:125]: I guess most customers would not have a node group already, so yes, good to remove that.

**Review by PJMODOS — APPROVED** [review]
This looks sane

**Comment by mwanner** [comment]
@djw-m Can I leave this up to you to merge at the best point in time?

**Comment by mwanner** [comment]
I resolved the merge conflict. Seems this needs an approval, again.

> You can. I assume there's nothing blocking this going out on your end?

@djw-m No, quite the opposite, this should have been part of the PGD 5.0.0 release. I.e. should better go out ASAP.

**Review by djw-m — APPROVED** [review]
LGTM - lets get this in

**Pull request merged**
